core/plannernotes.c

```diff
@@ -268,13 +295,29 @@ void add_plan_to_notes(struct diveplan *diveplan, struct dive *dive, bool show_d
 				/* Normally a gas change is displayed on the stopping segment, so only display a gas change at the end of
 				 * an ascent segment if it is not followed by a stop
 				 */
+
+				if(istrimix) {
+					deltaN2 = ((1000-newgasmix.he.permille-newgasmix.o2.permille)-(1000-lastprintgasmix.he.permille-lastprintgasmix.o2.permille))/10.0;
```

Comment:
According to our coding style rules there are many spaces missing everywhere in your code. This is one of the best examples:
`...1000 - newgas...`
Also there was one `if(...` which has to be a `if (...`
Funny, this currently feels very strange for me because Lubomir and the others taught this also to me only few months ago. I'm now just forwarding to you what I learned from the real pros ;-)